OpenRobertaServer/src/main/java/de/fhg/iais/roberta/persistence/dao/ProgramDao.java

```diff
@@ -253,9 +253,18 @@ public JSONArray loadGallery(int galleryId, int userId) {
             "where p.OWNER_ID = :galleryId " + //
             "  and p.ROBOT_ID = r.ID " + //
             "  and p.AUTHOR_ID = u.ID ";
+
+        if ( !robotGroup.isEmpty() ) {
+            galleryProgramSql = galleryProgramSql + "" + //
+                " and r.NAME = :robotGroup ";
```

Comment:
please try not to force formatting but use the installed roberta java formatter to aline the code